Runtime/Game/Requests/PurchaseRequest.cs

```diff
@@ -138,7 +143,7 @@ public static void AndroidPurchaseVerification(LootLockerAndroidPurchaseVerifica
             LootLockerServerRequest.CallAPI(endPoint.endPoint, endPoint.httpMethod, json, (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
         }
 
-        public static void PollOrderStatus(LootLockerGetRequest lootLockerGetRequest, Action<LootLockerCharacterLoadoutResponse> onComplete)
+        public static void PollOrderStatus(LootLockerGetRequest lootLockerGetRequest, Action<LootLockerPurchaseOrderStatus> onComplete)
```

Comment:
Should we instead mark this as deprecated?
Noone is probably using it, but just to be consistent.

Reply:
Done